.github/workflows/ci.yml

```diff
@@ -36,17 +36,22 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest]
         rust: [stable]
         node: [ '12', '14' ]
     steps:
       - name: Install Fluvio Local Cluster
-        uses: infinyon/fluvio@master
+        uses: simlay/fluvio@minikube-driver-fix
```

Comment:
Should probably merge infinyon/fluvio#579 for this.

Reply:
Can you turn #579 into non draft?